docker-compose.gcb.yml

```diff
@@ -85,6 +92,9 @@ services:
     depends_on:
       - zookeeper
     image: "${CLICKHOUSE_IMAGE:-ghcr.io/getsentry/image-mirror-altinity-clickhouse-server:23.3.19.33.altinitystable}"
+    hostname: clickhouse-query.local
+    extra_hosts:
+      - "clickhouse-query.local:127.0.0.1"  # Add entry to /etc/hosts file
```

Comment:
I suggest rewording the comment to explain why this is desirable, rather than restating what the code does.